### ISSUE TITLE
Cluster init (and possibly other jobs) occasionally fail to start

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1074]
Originally reported of Google Code with ID 1018.

```
There appears to be a pretty nasty race condition occurring immediately after the fork of a job, causing the job not to be started. The occurrence rate is pretty low, but it has been seen 2+ times during testing, enough to warrant concern. This behavior manifested itself after cluster init, but it could be happening in other situations as well.

Sample traceback of a crash:

Traceback (most recent call last):
  File "/usr/share/ganeti/2.13/ganeti/cli.py", line 1217, in GenericMain
    result = func(options, args)
  File "/usr/share/ganeti/2.13/ganeti/rpc/node.py", line 141, in wrapper
    return fn(*args, **kwargs)
  File "/usr/share/ganeti/2.13/ganeti/client/gnt_cluster.py", line 337, in InitCluster
    SubmitOpCode(op, opts=opts)
  File "/usr/share/ganeti/2.13/ganeti/cli.py", line 972, in SubmitOpCode
    reporter=reporter)
  File "/usr/share/ganeti/2.13/ganeti/cli.py", line 951, in PollJob
    return GenericPollJob(job_id, _LuxiJobPollCb(cl), reporter)
  File "/usr/share/ganeti/2.13/ganeti/cli.py", line 779, in GenericPollJob
    raise errors.OpExecError(str(msg))
OpExecError: job 1 failed to start: GenericError "ganeti job process input pipe: user error (Timeout in reading a response)"
```
Originally added on 2015-01-13 16:58:36 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
We have confirmed this issue can manifest itself in jobs other than the initial one.
```
Originally added on 2015-01-14 12:23:47 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-02-24 15:49:49 +0000 UTC.

Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-03-25 18:32:05 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:37:03 +0000 UTC.

Changed State: Released
